READMEs/datatokens-flow.md

```diff
@@ -25,11 +26,14 @@ cd barge
 # Clean up old containers (to be sure)
 docker system prune -a --volumes
 
-# Run barge: start ganache, Provider, Aquarius; deploy contracts; update ~/.ocean
-./start_ocean.sh
+# make sure to use the dev version of the operator service URL, to match the barge provider
+export OPERATOR_SERVICE_URL=https://c2d-dev.operator.oceanprotocol.com/
+
+#run barge: start ganache, Provider, Aquarius; deploy contracts; update ~/.ocean
+./start_ocean.sh  --with-provider2
```

Comment:
I have added 
`export OPERATOR_SERVICE_URL=https://c2d-dev.operator.oceanprotocol.com/` as it was used in the rest of quickstarts and doesn't affect quickstart

The same for  adding `--with-provider2`
`./start_ocean.sh --with-provider2`